lighthouse-core/scripts/lantern/download-traces.sh

```diff
@@ -27,9 +27,9 @@ fi
 
 rm -rf lantern-data/
 mkdir -p lantern-data/ && cd lantern-data
-echo $VERSION > version
 
 curl -o golden-lantern-traces.zip -L https://storage.googleapis.com/lh-lantern-data/golden-lantern-traces-$VERSION.zip
 
 unzip -q golden-lantern-traces.zip
 rm golden-lantern-traces.zip
+echo $VERSION > version
```

Comment:
minor thing, this is better in case the unzipping failing halfway bc user accidentally cancels.